File: ProfilesApi/Contracts/Responses/DoctorProfiles/GetDoctorAndPhotoProfilesResponse.cs
namespace ProfilesApi.Contracts.Responses.DoctorProfiles;

public class GetDoctorAndPhotoProfilesResponse
{
    public Guid Id { get; set; }
    public string FullName { get; set; }
    public string Specialization { get; set; }
    public int Experience { get; set; }
    public string Address { get; set; }
    public byte[] Photo { get; set; }
}

File: serverApp/src/app/ProfilesService/_models/responses/GetDoctorAndPhotoProfilesResponse.ts
import { DoctorStatusEnum } from "../DoctorStatusEnum";

export class GetDoctorAndPhotoProfilesResponse {
  constructor (
    public id: string,
    public fullName: string,
    public specializationName: string,
    public experience: number,
    public address: string,
    public photo: any
  ) {}
}
